doc/setup.md

```diff
@@ -41,6 +41,30 @@ PORT=3000 ASSETS_DEBUG=true ./script/start
 open http://localhost:3000
 ```
 
+## Seeding your dashboard with data
+
+Once you've authed locally, there will be a `user` record associated with your
+email address. There won't be much for you to see until your client slug is set,
+so find your user record and set it to `ncr`:
+
+```bash
+rails c
```

Comment:
Minor: use `bin/rails c` (and `bin/rake ...` below) so that the application starts faster (via [Spring](https://github.com/rails/spring)).